.github/workflows/tests.yaml

```diff
@@ -76,13 +76,15 @@ jobs:
                       .environment/gn_out/.ninja_log
                       .environment/pigweed-venv/*.log
             - name: Build Apps
-              timeout-minutes: 30
+              timeout-minutes: 40
```

Comment:
Does it take 10 more minutes of CI? Expanding the timeout this much is a lot

Reply:
No, I can move this down to 5 min